### PR TITLE
Bg user story 42 2

### PR DESCRIPTION
**Pull request by brian-greeson**

### CODE REVIEW

File: app/views/merchant/items/index.html.erb [hunk 1]
```diff
@@ -11,10 +11,10 @@
       <% end %>
       <% if item.active? %>
         <p>Active</p>
-        <%= link_to "Deactivate", merchant_items_update_path(item, active?: false), method: :patch %>
+        <%= link_to "Deactivate", merchant_item_update_path(item, active?: false), method: :patch %>
       <% else %>
         <p>Inactive</p>
-        <%= link_to "Activate", merchant_items_update_path(item, active?: true), method: :patch %>
+        <%= link_to "Activate", merchant_item_update_path(item, active?: true), method: :patch %>
```
Comment (kelshadarby) [inline app/views/merchant/items/index.html.erb:17]: Nice catch!

File: app/controllers/merchant/items_controller.rb [hunk 2]
```diff
@@ -7,9 +7,9 @@ def index
   def update
     item = Item.find(params[:id])
     if item.update(item_params)
-      flash[:success] = "Item #{item.id} is " + active_message(item.active?)
+      flash[:success] = "Item #{item.id} is " + item_active_message(item.active?)
     else
-      flash[:error] = @item.errors.full_messages.to_sentence
```
Comment (kelshadarby) [inline app/controllers/merchant/items_controller.rb:12]: Nice use of the model error messages

**Review by kelshadarby — APPROVED** [review]

File: app/views/admin/merchant/items/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  <% @merchant.items.each do |item| %>
+    <section class="grid-item" id='item-<%=item.id%>'>
+      <h2> <%=link_to item.name, "/items/#{item.id}" %> </h2>
+      <p>Sold by: <%=link_to item.merchant.name, "/merchants/#{item.merchant.id}" %></p>
+      <%=link_to image_tag(item.image), "/items/#{item.id}", id: "item-image-#{item.id}" %>
+      <p> <%= item.description %> </p>
+      <p>Price: <%=number_to_currency(item.price) %> </p>
+      <p>Inventory: <%= item.inventory %> </p>
+      <% if !@merchant %>
+      <% end %>
```
Comment (fredrondina) [inline app/views/admin/merchant/items/index.html.erb:11]: looks good, we will refactor this into a partial at a later date

File: app/views/merchant/items/index.html.erb [hunk 4]
```diff
@@ -11,10 +11,10 @@
       <% end %>
       <% if item.active? %>
         <p>Active</p>
-        <%= link_to "Deactivate", merchant_items_update_path(item, active?: false), method: :patch %>
```
Comment (fredrondina) [inline app/views/merchant/items/index.html.erb:14]: looks good, we will refactor this into a partial at a later date

**Review by fredrondina — APPROVED** [review]
fantastic work again

**Pull request merged**